cmd/dev_common.go

```diff
@@ -80,7 +81,11 @@ func addStackRegistryFlag(cmd *cobra.Command, flagVar *string, config *RootComma
 	defaultRegistry := getDefaultStackRegistry(config)
 	stackRegistryInConfigFile, err := getStackRegistryFromConfigFile(config)
 	if err != nil {
-		config.Debug.Logf("Error retrieving the stack registry from config file: %v", err)
+		if _, ok := err.(*NotAnAppsodyProject); ok {
```

Comment:
We could rechange this to only have an if

Reply:
I've changed this now. I'm not too sure about the boolean idea, I can give it a try later if we decide to go with that